marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/security/getAuthorities.sjs

```diff
@@ -15,14 +15,49 @@
  */
 'use strict';
 
-// TODO This is just stubbed out, a real implementation is needed.
-
-let response = {
-  "authorities": [
-    "canInstallDataHub",
-    "canReadLoadData",
-    "canWriteLoadData"
-  ]
+const Artifacts = require('/data-hub/5/artifacts/core.sjs');
+const roleIdToName = (roleID) => xdmp.roleName(roleID);
+const currentRoles = xdmp.getCurrentRoles().toArray().map(String);
+let currentRoleNames = currentRoles.map(roleIdToName);
+
+const defaultManageRoles = ['admin', 'data-hub-environment-manager'].map((roleName) => String(xdmp.role(roleName)));
+const manageRolesMustAllMatched = ['manage-admin', 'security'].map((roleName) => String(xdmp.role(roleName)));
+
+let hasManagerRole = defaultManageRoles.some((role) => currentRoles.includes(role));
+if (!hasManagerRole) {
+  hasManagerRole = manageRolesMustAllMatched.every((role) => currentRoles.indexOf(role) !== -1);
```

Comment:
This should be hasManageAdminRole.
With manageRolesMustAllMatched are we trying to check if a role has both manage-admin & security? If so, hasManageAdminAndSecurity sounds better to me.